lib/id3c/cli/redcap.py

```diff
@@ -576,6 +576,44 @@ def id(self) -> str:
             raise RuntimeError(f"Primary record id field «{self.project.record_id_field}» not available on fetched record") from e
 
 
+    def update_record(self, record: List[Dict[str, str]], date_format: str = "YMD", check_count: bool = True) -> int:
```

Comment:
Since this is updating only one redcap record, you do not need List[Dict[str, str]], specifically a list